client/web/src/nav/StatusMessagesNavItem.tsx

```diff
@@ -415,6 +424,7 @@ export class StatusMessagesNavItem extends React.PureComponent<Props, State> {
                     }
                     as={CloudOffOutlineIcon}
                     size="md"
+                    aria-hidden={true}
```

Comment:
```suggestion
                    title={tooltip}
```

Let's ensure the tooltip is attached as a label here. Once we have accessible tooltips this _might_ result in the label being accessed twice - but is we keep it hidden then screen readers won't even be able to trigger the tooltip

Reply:
If the tooltip is undefined, we should still set `aria-hidden`